test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -60,7 +60,7 @@ public string[] GetInstallCommandsFromLocal(string path)
                     "rhel" => new[]
                     {
                         "set -e",
-                        $"sudo rpm --nodeps -i {string.Join(' ', packages)}",
+                        $"sudo dnf -y localinstall {string.Join(' ', packages)}",
```

Comment:
The [DNF command reference](https://dnf.readthedocs.io/en/latest/command_ref.html#install-command) says that `localinstall` is a deprecated alias for `install`, so we should probably just use `install`.